cluster-management/setup/cluster-architectures/index.md

```diff
@@ -234,9 +234,13 @@ coreos:
   fleet:
     metadata: "role=worker,cabinet=two,disk=spinning"
     etcd_servers: "http://10.0.0.101:4001,http://10.0.0.102:4001,http://10.0.0.103:4001,http://10.0.0.104:4001,http://10.0.0.105:4001"
+  locksmith:
+    endpoint: "http://10.0.0.101:4001,http://10.0.0.102:4001,http://10.0.0.103:4001,http://10.0.0.104:4001,http://10.0.0.105:4001"
   units:
     - name: fleet.service
       command: start
+    - name: etcd.service
```

Comment:
you are correct. I would write the config like this instead:

```yaml
  units:
    - name: etcd.service
      mask: true
    - name: fleet.service
      command: start
```

This prevents etcd from ever running.